pkg/chunk/azure/blob_storage_client.go

```diff
@@ -17,11 +17,25 @@ import (
 	"github.com/cortexproject/cortex/pkg/util/flagext"
 )
 
+//blob service endpoint of AzureGlobal
 const blobURLFmt = "https://%s.blob.core.windows.net/%s/%s"
 const containerURLFmt = "https://%s.blob.core.windows.net/%s"
```

Comment:
Could you prefix these by `global`, please?

Reply:
done